tests/manage/monitoring/prometheusmetrics/test_rgw.py

```diff
@@ -0,0 +1,57 @@
+# -*- coding: utf8 -*-
+"""
+Test cases here performs Prometheus queries directly without a workload, to
+check that OCS Monitoring is configured and available as expected for RGW.
+"""
+
+import logging
+
+import pytest
+
+from ocs_ci.framework.testlib import skipif_ocs_version, tier4
+from ocs_ci.ocs import constants, defaults, ocp
+from ocs_ci.ocs import metrics
+from ocs_ci.ocs.resources.ocs import OCS
+from ocs_ci.utility.prometheus import PrometheusAPI
+
+
+logger = logging.getLogger(__name__)
+
+
+@skipif_ocs_version("<4.6")
+@tier4
```

Comment:
please add tier4a/b/c

Reply:
+1 (but I'm not sure into which one should it go, as I failed to find definition of tier4 suite a, b, c ...)